### PR TITLE
CI-53: React Gateway

### PR DESCRIPTION
**Pull request by LyubovP**
[CI-53](https://cloverpop-internship.atlassian.net/browse/CI-53)

Added gem 'jsonapi-serializer' to Gemfile

[CI-53]: https://cloverpop-internship.atlassian.net/browse/CI-53?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/controllers/home_controller.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class HomeController < ApplicationController  
+  def index
+  end
+  
+  def app
+  end
+end
```
Comment (VitaliiShevchenko) [inline app/controllers/home_controller.rb:7]: The same controller created by @SerhiiBorozenets in CI-34/CI-35

Comment (LyubovP) [reply app/controllers/home_controller.rb:7]: without it, I can't test the React settings + it is included in the React settings

File: app/javascript/controllers/index.js [hunk 2]
```diff
@@ -6,3 +6,6 @@ import { application } from "./application"
 
 import HelloController from "./hello_controller"
 application.register("hello", HelloController)
```
Comment (VitaliiShevchenko) [inline app/javascript/controllers/index.js:8]: Do we need in an Hello controller ? I think may to delete it

File: app/javascript/controllers/react_controller.js [hunk 3]
```diff
@@ -0,0 +1,13 @@
+import { Controller } from "@hotwired/stimulus"
+import React from "react";
+import { createRoot } from "react-dom/client";
+import App from "../components/App";
+
+// Connects to data-controller="react"
+export default class extends Controller {
+  connect() {
+    console.log("React controller connected");
```
Comment (rogergraves) [inline app/javascript/controllers/react_controller.js:9]: Please remove console logs if they aren't necessary

**Review by rogergraves — APPROVED** [review]
Look OK pending CodeClimate suggestions

**Pull request merged**